os/board/rtl8721csm/src/component/common/file_system/ftl/ftl.c

```diff
@@ -1546,6 +1549,8 @@ uint32_t ftl_secure_init(void)
 
 uint32_t ftl_init(uint32_t u32PageStartAddr, uint8_t pagenum)
 {
+    
+	device_mutex_lock(RT_DEV_LOCK_FLASH); 

```

Comment:
There is a return at line 1635. Don't you need to unlock before leaving?

Reply:
I have added the device_mutex_unlock.

Reply:
There is no change on each. Could you check them again?

Reply:
I have pushed again, please help to check.